### DISCUSSION TITLE
Add a way to set the default curve type of sliders on editor

### DISCUSSION THREAD

**Discussion by maotovisk** [ppy/osu#19896] [Ideas]
I've searched if anyone suggested this in issues/discussions and I didn't find anything, so sorry if something like this was suggested before.

I was talking to some friends while mapping and one of them said it would be really useful to have a way to set the default curve type of newly created sliders, it would be useful to make old style maps that use just beziers or gimmicks involving linear sliders, for example. 

A submenu option at the "Edit" menu called "Default slider curve type" would be a simple yet good UX imo.

**Comment by peppy** [comment]
I think being able to right click (or having a dropdown portion) to show a popover on each the hitobject create buttons would work quite well for these kinds of settings.

Would be even better if the current settings appeared in or next to the button. Will need some further experimentation, but should work better than filling up the menus with ruleset-specific settings.

**Reply by maotovisk** [reply]
That sounds like a good idea
